.env

```diff
@@ -0,0 +1,4 @@
+API_HOST=127.0.0.1 
+DATABASE_URL=postgres://docker:docker@localhost:5431/postgres 
+DEBUG=
+PORT=80
```

Comment:
Is this necessary/used?

Reply:
Was used for DOTENV now it's passed as a single DATABASE_URL in the package.json script. deletes dotenv dependency and the file. Thx!